### ISSUE TITLE
[1.12.2] Weird Bucket Texture

### ISSUE THREAD

**Issue by Jerome226** [the9grounds/AE-Additions#53]
Basically i'm using latest AE Additions (AEAdditions-1.12.2-1.3.4) and all buckets use the cell fluid texture.
![image](https://user-images.githubusercontent.com/55724525/149603626-688f3f7c-69d3-4096-95ad-40deded7b95d.png)

**Comment by Jerome226** [comment]
Also some cells from the mod seems to have buckets textures, weird issue.
![image](https://user-images.githubusercontent.com/55724525/149603901-cf8b650b-5403-466e-b4ad-3236bc1b51ea.png)

**Comment by Beachman4** [comment]
Yeah all those are left over from when I took over the mod. I'll get these removed with the next version
